pkg/kv/dynamodb/store.go

```diff
@@ -561,3 +581,10 @@ func (s *Store) StopPeriodicCheck() {
 		s.cancel = nil
 	}
 }
+
+func (s *Store) isSlowDownErr(err error) bool {
+	var errRequestLimitExceeded *types.RequestLimitExceeded
+	var errProvisionedThroughputExceededException *types.ProvisionedThroughputExceededException
+	var errThrottlingException *types.LimitExceededException
+	return errors.As(err, &errRequestLimitExceeded) || errors.As(err, &errProvisionedThroughputExceededException) || errors.As(err, &errThrottlingException)
```

Comment:
The v1 SDK had [IsErrorThrottle](https://github.com/aws/aws-sdk-go/blob/fdce8a5fc70227970e0adffd97f3e8edee308925/aws/request/retryer.go#L230).  Any chance that the v2 SDK will also have something like this?  Maybe [this](https://github.com/aws/aws-sdk-go-v2/blob/10c44870fbdabbbac25ffe1a28115efe4fd560d9/aws/retry/adaptive.go#L18-L24) will be relevant and do what we need?

My worry is that AWS may have a different idea than we do about what this list should be.

Reply:
Cool